src/lib/ngsi/ContextAttribute.cpp

```diff
@@ -40,6 +40,7 @@
 #include "rest/ConnectionInfo.h"
```

Comment:
I planned to use the old one ...

    - Add: q-string filters for compound values of attributes (Issue #1156)

If you prefer something like:

    - Add: q-string filters for compound values of attributes, both for queries and subscriptions (Issue #1156)

Then no probs ...